pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -248,36 +248,6 @@ var allFinishedState = PipelineRunState{{
 	},
 }}
 
-var oneTaskFailedToRetry = PipelineRunState{{
-	PipelineTask: &pts[3],
-	TaskRunName:  "pipelinerun-mytask-failed",
-	TaskRun:      makeFailed(trs[0]),
-	ResolvedTaskResources: &resources.ResolvedTaskResources{
-		TaskSpec: &task.Spec,
-	},
-}, {
-	PipelineTask: &pts[0],
-	TaskRunName:  "pipelinerun-mytask1",
-	TaskRun:      nil,
-	ResolvedTaskResources: &resources.ResolvedTaskResources{
-		TaskSpec: &task.Spec,
-	},
-}}
-var oneTaskFailedWithRetry = PipelineRunState{{
-	PipelineTask: &pts[3],
-	TaskRunName:  "pipelinerun-mytask-failed",
-	TaskRun:      makeRetried(*makeFailed(trs[0])),
-	ResolvedTaskResources: &resources.ResolvedTaskResources{
-		TaskSpec: &task.Spec,
-	},
-}, {
-	PipelineTask: &pts[0],
-	TaskRunName:  "pipelinerun-mytask1",
-	TaskRun:      nil,
-	ResolvedTaskResources: &resources.ResolvedTaskResources{
-		TaskSpec: &task.Spec,
-	},
-}}
```

Comment:
hm i think before we remove these we should check if this means we accidentally deleted some unit tests we didn't mean to?

Reply:
cc @joseblas as I think it comes from the retry feature recently merged :angel:

Reply:
It looks like it was never actually used in that PR. Not sure what it was meant for.